trie/trie.go

```diff
@@ -574,13 +580,12 @@ func (t *Trie) resolve(n node, prefix []byte) (node, error) {
 	return n, nil
 }
 
-func (t *Trie) resolveHash(n hashNode, prefix []byte, epoch uint16) (node, error) {
```

Comment:
I think this change was introduced in #84, is there a reason to remove it?

Reply:
It causes the compile error, so I remove it temporary, the https://github.com/node-real/bsc/pull/87 will refactor it too.